shaders/ScreenFilter.shader

```diff
@@ -0,0 +1,22 @@
+shader_type canvas_item;
+uniform int mode = 0;
+void fragment() {
+	vec4 px = texture(SCREEN_TEXTURE, SCREEN_UV);
+	if (mode==1)
```

Comment:
I think it would be nice to make the colours really messed up when `mode==0` to easily verify that the filter isn't on when not needed.